rust/alloc/boxed.rs

```diff
@@ -189,8 +189,6 @@ impl<T> Box<T> {
     /// ```
     #[cfg(not(no_global_oom_handling))]
     #[inline(always)]
-    #[doc(alias = "alloc")]
-    #[doc(alias = "malloc")]
```

Comment:
I think these aliases are deliberately added on ourside.

Reply:
I can unstage these changes, but it looked like they were removed between 1.54.0 and 1.55.0, presumably this commit https://github.com/rust-lang/rust/commit/618c805746dbbc874afa538f97fc5f643aa9f6dc

Reply:
Oh, my bad. We should remove them then.